tests/test_shortener.py

```diff
@@ -32,3 +32,31 @@ def test_regression_1_left_padding_with_zeros(self):
         shortener = UuidShortener()
 
         self.assertEqual(expected_uuid,  shortener.unshorten(shortened_uuid))
+
+
+class TestShortenedUuidGenerator(TestCase):
+
+    def test_one_way_shortener_with_prefix(self):
+        shortener = UuidShortener("dev")
+        short_id = ShortUuidGenerator(prefix="dev", uuid=uuid4)
+
+        short_uuid = short_id()
+
+        self.assertGreater(len(short_uuid), 4)
+        self.assertEqual(0, short_uuid.index("dev-"))
+        self.assertNotEqual(
```

Comment:
Can this assert be made more deterministic? 
For example, the uuid function can be mocked to return a specific uuid that we know the shortened version of and we can assert against that.
wdyt?